https/README.md

```diff
@@ -49,3 +79,40 @@ hash. A higher cost will en up slowing down the authentication process.
 Depending on the machine, a cost of 10 will take about ~70ms where a cost of
 18 can take up to a few seconds. That hash will be computed on every
 password-protected request.
+
+
+## Available cipher suites
```

Comment:
I'd much prefer we point to the Go docs rather than duplicating this, it's one less thing to maintain.

Reply:
done